tensorflow/training/docker/2.8/py3/example/Dockerfile.gpu

```diff
@@ -1,4 +1,4 @@
-# Expecting base image to be the image built by ../cu112/Dockerfile.e3.gpu
+# Expecting base image to be the image built by ../cu112/Dockerfile.gpu e3 target
```

Comment:
Just wanted to confirm this. E3 images act as the base for example images?

Reply:
yes, e3 gpu images are the base for example images